Structure_Engine/Create/Loads/GeometricalLineLoad.cs

```diff
@@ -57,6 +57,37 @@ public static GeometricalLineLoad GeometricalLineLoad(Line location, Loadcase lo
             };
         }
 
+        [Description("Creates a GeometricalLineLoad with a constant force across the length of the Line.")]
+        [InputFromProperty("location")]
+        [InputFromProperty("loadcase")]
+        [Input("force", "The force to be applied to the full length of the Line.", typeof(BH.oM.Quantities.Attributes.Force))]
+        [Input("panels", "The Panels to apply the GeometricalLineLoad to.")]
+        [Input("name", "The name of the created load.")]
+        [Output("geoLineLoad", "The created GeometricalLineLoad.")]
+        public static GeometricalLineLoad GeometricalLineLoad(Line location, Loadcase loadcase, Vector moment = null, Vector force = null, IEnumerable<Panel> panels = null, string name = "")
```

Comment:
```suggestion
        [Input("force", "The force to be applied to the full length of the Line.", typeof(BH.oM.Quantities.Attributes.Force))]
        [Input("moment", "The momentto be applied to the full length of the Line.", typeof(BH.oM.Quantities.Attributes.Moment))]
        [Input("objects", "The IAreaElement (i.e. Panels or FEMesh) to apply the GeometricalLineLoad to.")]
        [Input("name", "The name of the created load.")]
        [Output("geoLineLoad", "The created GeometricalLineLoad.")]
        public static GeometricalLineLoad GeometricalLineLoad(Line location, Loadcase loadcase, Vector force= null, Vector moment= null, IEnumerable<IAreaElement> objects = null, string name = "")
```

It's a bit odd from a user perspective that the moment comes before the force.

I know this is being written in mind for the RFEM_Toolkit, but we do have the `IAreaElement` interface for a reason, so if in the future a toolkit that implement `FEMesh` wants to incorperate `GeometricalLineLoad`, it would make sense that they do not need to write a new method.